docs/database/gravity/lists.md

```diff
@@ -14,16 +15,22 @@ Label | Type | Uniqueness enforced | Content
 `date_modified` | integer | No | Timestamp when domain was last modified, automatically updated when a record is changed
 `comment` | text | No | Optional field for arbitrary user comments
 
-## Blacklist Table (`blacklist`)
-The `blacklist` table contains all blacklisted domains. Just like the `whitelist` table, it has a few extra fields to store data related to a given domain such as the `enabled` state, the dates when the domain were added and when it has last been modified, and an optional comment.
+### RegEx Whitelisting Table (`regex_whitelist`)
```

Comment:
For consistency, do we want to use `Regex` or `RegEx`? Up to now it seemed that `Regex` is the most common form.